arangograph/single-sign-on.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+description: Single Sign-On (SSO) and SCIM
+title: Single Sign-On (SSO)
+---
+# Single Sign-On (SSO)
+
+ArangoGraph provides support to enable Single Sign-On (SSO) authentication using
+SAML with SCIM provisioning. This enables you to propagate to ArangoGraph any
+user access changes by using the dedicated API.
+
+[SCIM](https://www.rfc-editor.org/rfc/rfc7644){:target="_blank"}, or the System
```

Comment:
IMO (later) should introduce two sections, one for SAML, and one for SCIM.

Also just FYI we have plans to introduce sample projects and also publish it to the Okta marketplace.